docs/devices/AB3257001NJ.md

```diff
@@ -14,6 +14,7 @@ description: "Integrate your OSRAM AB3257001NJ via Zigbee2MQTT with whatever sma
 | Description | Smart+ plug |
 | Exposes | switch (state), linkquality |
 | Picture | ![OSRAM AB3257001NJ](../images/devices/AB3257001NJ.jpg) |
```

Comment:
Same goes for https://github.com/Koenkk/zigbee2mqtt.io/pull/816/files, when searching for this model I only find this product with an OSRAM label (not ledvance).

https://www.google.com/search?q=AB3257001NJ&safe=off&rlz=1C5CHFA_enNL959NL959&sxsrf=ALeKk0157cFoePb6oECFm2sUMP30aMkFxw:1625512015692&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjdzJeS0MzxAhVJi6QKHUWkB8A4ChD8BSgCegQIARAE&biw=1440&bih=821

Reply:
One of three that I own, all labeled Ledvance on the product and box.

![PXL_20210705_193902064.jpg](https://user-images.githubusercontent.com/1300141/124512923-20007d00-ddc9-11eb-91ec-d00204f8dcaf.jpg)

![Screenshot_20210705-214026.png](https://user-images.githubusercontent.com/1300141/124512935-268ef480-ddc9-11eb-9efc-26d63986feaa.png)

Reply:
Can you share the data/database.db entries for all 3? (I'm wondering if the `manufName` differs)

Reply:
My LEDVANCE branded AC10691:
```
{"id":33,"type":"Router","ieeeAddr":"0xf0d1b8000014f33d","nwkAddr":32152,"manufId":4489,"manufName":"LEDVANCE","powerSource":"Mains (single phase)","modelId":"Plug Z3","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":266,"inClusterList":[0,3,4,5,6,2821,4096,64527],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"Plug Z3","manufacturerName":"LEDVANCE","powerSource":1,"zclVersion":2,"appVersion":1,"stackVersion":2,"hwVersion":1,"dateCode":"Dec 10 2018\u0000\u0000\u0000\u0000\u0000","swBuildId":"00103101"}},"genOnOff":{"attributes":{"onOff":1}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00212effff06471c","endpointID":1}],"configuredReportings":[{"cluster":6,"attrId":0,"minRepIntval":0,"maxRepIntval":3600,"repChange":0}],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":1,"stackVersion":2,"hwVersion":1,"dateCode":"Dec 10 2018\u0000\u0000\u0000\u0000\u0000","swBuildId":"00103101","zclVersion":2,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1625519255751}
```

My LEDVANCE branded AB3257001NJ:
```
{"id":36,"type":"Router","ieeeAddr":"0x7cb03eaa0a0befab","nwkAddr":16257,"manufId":48042,"manufName":"OSRAM","powerSource":"Mains (single phase)","modelId":"Plug 01","epList":[3],"endpoints":{"3":{"profId":49246,"epId":3,"devId":16,"inClusterList":[4096,0,3,4,5,6,2820,64527],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"Plug 01","manufacturerName":"OSRAM","powerSource":1,"zclVersion":1,"appVersion":9,"stackVersion":2}},"genOnOff":{"attributes":{"onOff":1}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00212effff06471c","endpointID":1}],"configuredReportings":[{"cluster":6,"attrId":0,"minRepIntval":0,"maxRepIntval":3600,"repChange":0}],"meta":{}}},"appVersion":9,"stackVersion":2,"zclVersion":1,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1625516788137}
{"id":37,"type":"Router","ieeeAddr":"0x7cb03eaa0a0bc9ea","nwkAddr":24615,"manufId":48042,"manufName":"OSRAM","powerSource":"Mains (single phase)","modelId":"Plug 01","epList":[3],"endpoints":{"3":{"profId":49246,"epId":3,"devId":16,"inClusterList":[4096,0,3,4,5,6,2820,64527],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"Plug 01","manufacturerName":"OSRAM","powerSource":1,"zclVersion":1,"appVersion":9,"stackVersion":2,"hwVersion":1,"dateCode":"20140331DEOS****","swBuildId":"V1.05.09"}},"genOnOff":{"attributes":{"onOff":0}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00212effff06471c","endpointID":1}],"configuredReportings":[{"cluster":6,"attrId":0,"minRepIntval":0,"maxRepIntval":3600,"repChange":0}],"meta":{}}},"appVersion":9,"stackVersion":2,"hwVersion":1,"dateCode":"20140331DEOS****","swBuildId":"V1.05.09","zclVersion":1,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1625518106455}
{"id":38,"type":"Router","ieeeAddr":"0x7cb03eaa0a0bcb45","nwkAddr":23759,"manufId":48042,"manufName":"OSRAM","powerSource":"Mains (single phase)","modelId":"Plug 01","epList":[3],"endpoints":{"3":{"profId":49246,"epId":3,"devId":16,"inClusterList":[4096,0,3,4,5,6,2820,64527],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"Plug 01","manufacturerName":"OSRAM","powerSource":1,"zclVersion":1,"appVersion":9,"stackVersion":2,"hwVersion":1,"dateCode":"20140331DEOS****","swBuildId":"V1.05.09"}},"genOnOff":{"attributes":{"onOff":0}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00212effff06471c","endpointID":1}],"configuredReportings":[{"cluster":6,"attrId":0,"minRepIntval":0,"maxRepIntval":3600,"repChange":0}],"meta":{}}},"appVersion":9,"stackVersion":2,"hwVersion":1,"dateCode":"20140331DEOS****","swBuildId":"V1.05.09","zclVersion":1,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1625518752832}
```

Reply:
The AC10691 is already separated: https://www.zigbee2mqtt.io/devices/AC10691.html , does the image match yours?

Reply:
Yes, seems the device is exactly the same.